include/charmlite/core/common.hpp

```diff
@@ -115,97 +123,48 @@ namespace cmk {
         }
     };
 
-    /* general terminology :
- * - kind refers to a family of instances
- * - id/index refers to a specific instance
- */
+    using entry_table_t = std::vector<entry_record_>;
+    using entry_id_t = typename entry_table_t::size_type;
 
-    // TODO ( rename to callback_fn_t )
-    template <typename Message>
-    using callback_fn_t = void (*)(message_ptr<Message>&&);
-    using callback_table_t = std::vector<callback_fn_t<message>>;
-    using callback_id_t = typename callback_table_t::size_type;
+    enum class destination_kind : std::uint8_t
+    {
+        kInvalid = 0,
```

Comment:
We can probably the `k` prefix since these are enclosed within a class now. It's a holdover from the Google style guide, used to indicate global variables.

Reply:
Sure, I can get rid of them.